src/Lime.Protocol.UnitTests/DocumentExtensionsTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using Lime.Messaging.Contents;
+using NUnit.Framework;
+using Shouldly;
+
+namespace Lime.Protocol.UnitTests
+{
+    [TestFixture]
+    public class DocumentExtensionsTests
+    {
+        [Test]
+        [Category("ToDocumentContainer")]
+        public void Select_ToContainer()
+        {
+            // Arrange
+            var select = new Select();
+
+            // Act
+            var container = select.ToDocumentContainer();
+
+            // Assert
+            container.Value.ShouldBe(select);
+            container.Type.ShouldBe(Select.MediaType);
+        }
```

Comment:
All the test cases have the same logic: build a `Document`, create a `DocumentContainer` from it, and check the `Value` and `Type` properties.
You can use [TestCaseData](https://github.com/nunit/docs/wiki/TestCaseData) to return the documents and put the "Act" and "Assert" steps on a single method.